app/scripts/lib/ens-ipfs/resolver.js

```diff
@@ -56,22 +54,3 @@ async function resolveEnsToIpfsContentId ({ provider, name }) {
 function hexValueIsEmpty (value) {
   return [undefined, null, '0x', '0x0', '0x0000000000000000000000000000000000000000000000000000000000000000'].includes(value)
 }
-
-function getRegistryForChainId (chainId) {
-  switch (chainId) {
-    // mainnet
-    case 1:
-      return '0x314159265dd8dbb310642f98f50c066173c1259b'
-    // ropsten
-    case 3:
-      return '0x112234455c3a32fd11230c42e7bccd4a84e02010'
-    // rinkeby
-    case 4:
-      return '0xe7410170f87102df0055eb195163a03b7f2bff4a'
-    // goerli
-    case 5:
-      return '0x112234455c3a32fd11230c42e7bccd4a84e02010'
-    default:
-      return null
```

Comment:
I think this case was important as well. We want to resolve to the same address for all _known_ networks with ENS deployments (mainnet and the testnets), but I'm not sure we should use it for _all_ networks

Reply:
Hmm, fair, I can add this case back